src/Form/Mapper/FormMapper.php

```diff
@@ -21,56 +21,30 @@
 interface FormMapper
 {
     /**
-     * @param string $name
-     * @param mixed  $type
-     *
-     * @return FormBuilderInterface
+     * @param mixed $type
      */
-    public function create($name, $type = null, array $options = []);
+    public function create(string $name, ?string $type = null, array $options = []): FormBuilderInterface;
 
     /**
-     * @param array $keys field names
-     *
-     * @return self
+     * @param string[] $keys field names
      */
-    public function reorder(array $keys);
+    public function reorder(array $keys): self;
 
     /**
      * @param FormBuilderInterface|string $name
-     * @param string                      $type
-     *
-     * @return self
      */
-    public function add($name, $type = null, array $options = []);
+    public function add($name, ?string $type = null, array $options = []): self;
 
-    /**
-     * @param string $key
-     *
-     * @return self
-     */
-    public function remove($key);
+    public function remove(string $key): self;
 
-    /**
-     * @return self
-     */
-    public function setHelps(array $helps = []);
+    public function setHelps(array $helps = []): self;
 
-    /**
-     * @return self
-     */
-    public function addHelp($name, $help);
+    public function addHelp($name, $help): self;
```

Comment:
I don't know any implementation of this interface, but I'm pretty sure these arguments are always strings. At least is what happen at [`Sonata\AdminBundle\Form\FormMapper::addHelp()`](https://github.com/sonata-project/SonataAdminBundle/blob/bf184d36a26fe47e6b0db2c8e18d03bbd9cbcab4/src/Form/FormMapper.php#L256).
```suggestion
    public function addHelp(string $name, string $help): self;
```

Reply:
@core23 you wrote that interface, what do you think? Sounds reasonable…

Reply:
They are always string :+1: